lib/bolt/bolt_option_parser.rb

```diff
@@ -189,7 +189,7 @@ def initialize(options)
       end
       define('--boltdir FILEPATH',
              'Specify what Boltdir to load config from (default: autodiscovered from current working dir)') do |path|
-        @options[:configfile] = path
+        @options[:boltdir] = path
       end
       define('--configfile FILEPATH',
              'Specify where to load config from (default: ~/.puppetlabs/bolt/bolt.yaml)') do |path|
```

Comment:
What if we had `--configfile` directly set `:boltdir`? I'm in favor of fewer options to handle later.

Reply:
As I read more, this wouldn't really make sense. configfile can have a different name, and the logic around loading them is a little different.